darwin/client.py

```diff
@@ -714,6 +714,73 @@ def reset_item(self, dataset_slug: str, team_slug: str, payload: Dict[str, Any])
         """
         self._put_raw(f"teams/{team_slug}/datasets/{dataset_slug}/items/reset", payload, team_slug)
 
+    def post_workflow_comment(
+        self, workflow_id: int, text: str, x: float = 1, y: float = 1, w: float = 1, h: float = 1
+    ) -> int:
+        """
+        Creates a comment box with the given text for the given workflow.
+
+        Parameters
+        ----------
+        workflow_id: int
+            The id of the workflow that will receive the comment.
+        text: str
+            The comment itself. 
+        x: float, default: 1
+            The top left X coordinate value of the comment box.
+        y: float, default: 1
+            The top left Y coordinate value of the comment box.
+        w: float, default: 1
+            The width of the comment box.
+        h: float, default: 1
+            The height of the comment box.
+
+        Returns
+        -------
+        int
+            The id of the created comment.
+        """
+        response: Dict[str, Any] = cast(
```

Comment:
Just to be 100% sure here, why do we need `cast` here?

Reply:
Because `_post` returns a Union type.